py3dtilers/IfcTiler/IfcTiler.py

```diff
@@ -60,14 +65,20 @@ def main():
 
     this function creates an ifc tileset handling one ifc classe per tiles
     """
+    logging.basicConfig(filename='ifctiler.log', level=logging.INFO,filemode="w")
+    start_time = time.time()
+    logging.info('Started')
     ifc_tiler = IfcTiler()
     ifc_tiler.parse_command_line()
     args = ifc_tiler.args
+    args.offset = np.array(args.offset,dtype=np.float32)
+
     tileset = ifc_tiler.from_ifc(args.file_path, args.grouped_by, args.originalUnit, args.targetedUnit)
 
     if(tileset is not None):
+        tileset.get_root_tile().set_bounding_volume(BoundingVolumeBox())
```

Comment:
The tileset's bounding box is already set in [tileset_creation.py](https://github.com/VCityTeam/py3dtilers/blob/master/py3dtilers/Common/tileset_creation.py#L43), before we return the tileset. This line can be deleted